content/articles/london-nixos-meetup-1.md

```diff
@@ -0,0 +1,14 @@
+Title: First London NixOS meetup
+Date: 2015-03-26
+Short_summary: We are organising a NixOS meetup!
+Category: Events
+Authors: Vincent
+
+Hey everyone,  
+
+We are organising a NixOs meetup in London in [Mozilla offices](https://goo.gl/maps/EyM1r) the 21st of April at 7pm. 
```

Comment:
"the 21st of April at 7pm, in space generously provided by Mozilla London (it's a really cool space!)"

(to say thanks to Mozilla, that's the custom)